### ISSUE TITLE
IsActiveTeamMemberInputFilter is instanciated twice, wrong default highlighting

### ISSUE THREAD

**Issue by juergmessmer** [rapila/plugin-schulcms#3]
If admin/team_members list is called the is_active_team_member input filter is by default set to false and all team members are displayed.
The query is done correctly but the checkbox is checked. Trying to debug shows that the input filter is instanciated twice. The first time correctly with default value "false", the second time with "true".

How to fix this?

**Comment by sabberworm** [comment]
Could one time be on the detail, the other on the list filter?

**Comment by juergmessmer** [comment]
No, it's just a filter for the team member list and only used there
